packages/core/components/InputOrGroup/fields/ObjectField/styles.module.css

```diff
@@ -0,0 +1,33 @@
+/**
+ * ObjectField
+ */
+
+.ObjectField {
+  display: flex;
+  flex-direction: column;
+  background-color: white;
+  border: 1px solid var(--puck-color-grey-8);
+  border-radius: 4px;
+}
+
+/**
+ * ObjectFieldItem
+ */
+
+.ObjectFieldItem {
+  background: white;
+  border-top-left-radius: 4px;
+  border-top-right-radius: 4px;
+  display: block;
+  margin-bottom: 1px; /* Simulate border with margin so we can hide it whilst dragging */
+
+  border-bottom: 0;
+  outline-offset: 0px !important; /* Important helps to override Nextra docs */
+  outline: 1px solid var(--puck-color-azure-6) !important; /* Important helps to override Nextra docs */
```

Comment:
These styles are not being applied, because the component never uses `getClassNameItem()`. In the docs site the CSS module class is `undefined`. So perhaps this is the issue, and these `!important` aren't needed?

Reply:
<img width="377" alt="Screenshot 2023-11-27 at 12 05 59" src="https://github.com/measuredco/puck/assets/98794/280f1e28-67dc-48c4-b8d1-1f8c61c2ebf8">

Reply:
Good catch! Lazy copy pasta from array fields. I removed the whole concept of the `ObjectFieldItem` here, which included these unused styles.